src/Microsoft.Azure.WebJobs.Host/JobHost.cs

```diff
@@ -85,6 +85,14 @@ public JobHost(JobHostConfiguration configuration)
             _stoppingTokenSource = CancellationTokenSource.CreateLinkedTokenSource(_shutdownTokenSource.Token);
         }
 
+        /// <summary>
+        /// Gets the JobHostConfiguration of this JobHost
+        /// </summary>
+        public JobHostConfiguration Configuration
```

Comment:
> public JobHostConfiguration Configuration [](start = 7, length = 42)

Adding this also breaks layering. If we fix the other JobHost.Call issue, hopefully we can then avoid this.